rpa-manager/modules/ROOT/pages/processautomation-prepare-deployment-credential.adoc

```diff
@@ -0,0 +1,179 @@
++
+The host address used to get the credentials. The default address for Outlook services is `https://login.microsoftonline.com/common/oauth2/v2.0/`. This address might change if the customer uses a self hosted service.
+*** *Client ID*
++
+To authenticate with the OAuth Host, an Azure application must be defined in the customer's space. This Azure application has an ID, which is unique in the entire OAuth Host space, and that allows users to explicitly grant or revoke access for this Azure application to their accounts.
++
+For more information about how to create this Azure application for Outlook, see https://learn.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app[Quickstart: Register an application with the Microsoft identity platform^].
+*** *Client Secret*
++
+This secret enables RPA Manager to prove to the OAuth Host that it received the permission to access user accounts on behalf of the registered Azure application. Without it, the authentication is not possible.
+*** *Redirect URI*
++
+The URI to which the OAuth Host redirects after the user completes the authentication attempt. This redirect URI must be registered with the Azure application. The OAuth Host allows the authentication only if the entered redirect URI matches one of the registered URIs.
+*** *Scopes*
++
+The scopes define which permission the user grants to the Azure application. Grant the following permissions:
++
+**** `offline_access`
++
+(*Required*) Enables access via a refresh token, which you can use to repeatedly log in to the mail services without requiring user interaction each time.
+**** `https://outlook.office.com/IMAP.AccessAsUser.All`
++
+(*Required* when using IMAP) Enables reading and moving emails from the Outlook IMAP server.
+**** `https://outlook.office.com/POP.AccessAsUser.All`
++
+(*Required* when using POP3) Enables reading emails from the Outlook POP3 server.
+**** `https://outlook.office.com/SMTP.Send`
++
+(*Required* when using SMTP) Enables sending emails from the Outlook SMTP server.
+**** `openid email`
++
+Enables RPA Manager to automatically detect the email account used to log in to the Azure application. If this scope is omitted, you must provide an email in RPA Builder.
++
+*** *E-Mail Address*
++
+Email address for accessing Outlook.
+*** *Refresh Token*
++
+Refresh token generated with the authentication properties.
++
+Learn how to xref:rpa-builder::toolbox-variable-handling-credentials-for-oauth.adoc#generating-an-oauth-refresh-token[generate an OAuth refresh token with RPA Builder].
+** *User Account Credentials*
++
+Use this credential for accessing all other kinds of applications. Fill in the the username and password.
+. Click *OK*.
+
+== Check the Usage of a Credential
+
+Before editing a credential, check its usage to avoid inadvertent side-effects. You can only check the usage a credential if you belong to the <<form-create-credential-useraffiliation, affiliated users>> of that credential.
```

Comment:
```suggestion
Before editing a credential, check its usage to avoid inadvertent side effects. You can check the usage of a credential only if you belong to the <<form-create-credential-useraffiliation, affiliated users>> of that credential.
```